composer.json

```diff
@@ -30,13 +30,49 @@
         }
     },
     "extra": {
-        "class": "ComposerGpgVerify\\Verifier",
+        "class": "ComposerGpgVerify\\Verify",
         "branch-alias": {
             "dev-master": "2.0.x-dev"
+        },
+        "composer-gpg-verify": {
+            "allow-unsigned": [
+                "composer/ca-bundle",
+                "composer/composer",
+                "composer/semver",
+                "composer/spdx-licenses",
+                "psr/log",
+                "justinrainbow/json-schema",
+                "seld/cli-prompt",
+                "seld/jsonlint",
+                "seld/phar-utils",
+                "symfony/console",
+                "symfony/debug",
+                "symfony/filesystem",
+                "symfony/finder",
+                "symfony/polyfill-mbstring",
+                "symfony/process"
+            ],
+            "allow-untrusted": [
+                "composer/ca-bundle",
+                "composer/composer",
+                "composer/semver",
+                "composer/spdx-licenses",
+                "psr/log",
+                "justinrainbow/json-schema",
+                "seld/cli-prompt",
+                "seld/jsonlint",
+                "seld/phar-utils",
+                "symfony/console",
+                "symfony/debug",
+                "symfony/filesystem",
+                "symfony/finder",
+                "symfony/polyfill-mbstring",
+                "symfony/process"
+            ]
         }
     },
     "scripts": {
-        "post-update-cmd":  "PackageVersions\\Verifier::verifyDownloadedTags",
-        "post-install-cmd": "PackageVersions\\Verifier::verifyDownloadedTags"
+        "post-update-cmd":  "ComposerGpgVerify\\Verify::verify",
+        "post-install-cmd": "ComposerGpgVerify\\Verify::verify"
```

Comment:
why using scripts for that ? this applies only to your own CI. This should be handled by the plugin API, no ?